afdko/Tools/Programs/makeotf/makeotf_lib/source/hotconv/cmap.c

```diff
@@ -719,64 +719,125 @@ static Format2 *makeFormat2(cmapCtx h) {
 
 /* ------- end format 2 */
 
+static void setOrdered( Mapping *mapping, int codeBreak, int span, int index)
+{
+    int threshold;
+    /* If the ordered segment starts with a code break or ends with a code break,
+     then the cost of keeping this as a separate segment is just one new segment break
+     (the following segment break).
+     Else, the cost is adding two new segment breaks - the one to start this segment,
+     and to start the next segment. We mark as NOT orderd those segments that are no larger
+     than the threshold.
+     */
+    if ( (mapping[index-span].flags & CODE_BREAK) || codeBreak)
+        threshold = 4;
+    else
+        threshold = 8;
+    if (span > threshold)
+        mapping[index - span].ordered = 1;
+
+}
+
 /* Partition mappings optimally for format 4. Assumes mapping in code order */
 static int partitionRanges(cmapCtx h, Mapping *mapping) {
 	int span;
 	int ordered;
-	int overflow;
+    int lastOrdered;
```

Comment:
seems to be a mix of tabs vs spaces here, please make the indentation consistent